tools/disasm/disasm.py

```diff
@@ -91,6 +91,8 @@ def proper_name(symbol, in_data=False, is_symbol=True):
     # hacks
     if symbol == 0x809C46F0: # ovl_En_Encount4 fake symbol at the very end of the data section
         return variables_ast[0x809C46DC][0] + " + 0x14"
+    elif symbol == 0x801EF66D: # z_message_nes constant-folding stray fairy array
+        return variables_ast[0x801EF670][0] + f" - 0x{0x801EF670 - 0x801EF66D:X}"
```

Comment:
What's this? :o

Reply:
This is Thar0's solution to the fake bss that was in collision_check.c that was put in shawlucas' PR. It just so happened to be the solution I needed as well to fix my bss problems so I grabbed it from that PR